shim.c

```diff
@@ -798,3 +798,7 @@ long X_X509_get_version(const X509 *x) {
 int X_X509_set_version(X509 *x, long version) {
 	return X509_set_version(x, version);
 }
+
+const char *get_default_cert_dir(void) {
+  return X509_get_default_cert_dir();
+}
```

Comment:
I am not sure if formatting is consistent with the rest of the file.

Reply:
Well, it is consistent with the functions above it. But the indentation is not consistent at all throughout the file, changing from 2 to 4 indents :/